modules/images-configuration-parameters-hcp.adoc

```diff
@@ -0,0 +1,43 @@
+====
+
+[cols="3a,8a",options="header"]
+|===
+|Parameters for ROSA CLI |Description
+
+|`registry-config-allowed-registries`
+|Registries for which image pull and push actions are allowed. To specify all subdomains, add the asterisk (`\*`) wildcard character as a prefix to the domain name. For example, `*.example.com`. You can specify an individual repository within a registry. For example: `reg1.io/myrepo/myapp:latest`. All other registries are blocked. The format should be a comma-separated list of allowed registries. For example: `allowed.io, allowed.io2`
+
+|`registry-config-insecure-registries`
+|Registries which do not have a valid TLS certificate or only support HTTP connections. To specify all subdomains, add the asterisk (`\*`) wildcard character as a prefix to the domain name. For example, `*.example.com`. You can specify an individual repository within a registry. For example: `reg1.io/myrepo/myapp:latest`. The format should be a comma-separated list of insecure registries. For example: `insecure.io, insecure.io2`
+
+|`registry-config-blocked-registries`
+|Registries for which image pull and push actions are denied. To specify all subdomains, add the asterisk (`\*`) wildcard character as a prefix to the domain name. For example, `*.example.com`. You can specify an individual repository within a registry. For example: `reg1.io/myrepo/myapp:latest`. All other registries are allowed. The format should be a comma-separated list of blocked registries. For example: `blocked.io, blocked.io2`
+
+|`registry-config-allowed-registries-for-import`
+|Contains configuration that determines how the container runtime should treat individual registries when accessing images for builds and pods. For example, whether or not to allow insecure access. It does not contain configuration for the internal cluster registry. Limits the container image registries from which normal users can import images. The format should be a comma-separated list of 'domainName:insecure'. 'domainName' specifies a domain name for the registry. 'insecure' indicates whether the registry is secure or insecure.
+
+|`registry-config-additional-trusted-ca`
+|A json file containing the registry hostname as the key, and the PEM-encoded certificate as the value, for each additional registry CA to trust.
+
+|`registry-config-platform-allowlist`
+|A list of Red{nbsp}Hat registries is automatically allowed. This list can be periodically updated and impacted clusters will receive a notification with the new allowlist id. In such case, user needs to use this parameter to update to the expected id to the expected one.
+
+|===
+
+[WARNING]
+====
+When the `allowedRegistries` parameter is defined, all registries, are blocked unless explicitly listed. To prevent pod failure, a list of Red Hat registries is automatically whitelisted, as they are required by payload images within your environment. For disconnected clusters, mirror registries should be added by the user itself.
```

Comment:
```suggestion
When the `allowedRegistries` parameter is defined, all registries, are blocked unless explicitly listed. To prevent pod failure, a list of Red Hat registries is automatically whitelisted, as they are required by payload images within your environment. The current list is composed by `image-registry.openshift-image-registry.svc:5000,quay.io,registry.redhat.io` and it is also visible when running `rosa describe cluster`. 
```

Reply:
>The current list is composed by

should become

> the current list consists of

Reply:
Changed in both places to "The current list consists of `image-registry.openshift-image-registry.svc:5000,quay.io,registry.redhat.io` and it is also visible when running `rosa describe cluster`."